### PR TITLE
<SelectableAccordion/> - migrate to use <Radio/>

### PR DESCRIPTION
**Pull request by Zivilekin**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6702

File: packages/wix-style-react/src/SelectableAccordion/Item/Item.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { WixStyleReactContext } from '../../WixStyleReactProvider/context';
 import Checkbox from '../../Checkbox';
 import ToggleSwitch from '../../ToggleSwitch';
-import RadioGroup from '../../RadioGroup';
+import Radio from '../../Radio/Radio';
```
Comment (EgoziE) [inline packages/wix-style-react/src/SelectableAccordion/Item/Item.js:6]: A component should be imported from the index
```suggestion
import Radio from '../../Radio';
```

**Review by mykas — APPROVED** [review]
Wow! Smooth transition :D

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6702/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6702

**Pull request merged**